cluster.yml

```diff
@@ -126,7 +126,7 @@
   roles:
     - { role: kubespray-defaults}
     - { role: dnsmasq, when: "dns_mode == 'dnsmasq_kubedns'", tags: dnsmasq }
-    - { role: kubernetes/preinstall, when: "dns_mode != 'none' and resolvconf_mode == 'host_resolvconf'", tags: resolvconf }
+    - { role: kubernetes/preinstall, when: "dns_mode != 'none' and resolvconf_mode == 'host_resolvconf'", tags: resolvconf, dns_late: true }
```

Comment:
tags with value ?

Reply:
It sets a variable dns_late to true. Look at lines 65 and 71 in cluster.yml

Reply:
ah i misread line, there is comma after tags :)